docs/topics/changing_user_preferences.adoc

```diff
@@ -14,3 +14,5 @@ include::modules/adding_github_org.adoc[leveloffset=+1]
 include::modules/opting_in_to_features.adoc[leveloffset=+1]
 
 include::modules/viewing_resource_usage.adoc[leveloffset=+1]
+
+include::modules/reviewing_detailed_resource_information.adoc[leveloffset=+1]
```

Comment:
Imho this module seems misplaced here, the user would have no context for this and it does not belong to the Settings or Profile sections which the assembly changing_user_preferences deals with.
I suggest we:
1. Create an Assembly called 'Monitoring your resources' and add modules viewing_resource_usage and reviewing_detailed_resource_information to it. 
2. Add it after the 'Working with Che workspaces' assembly, this will provide a logical work flow and context to the users.
Wdyt?

Reply:
It's a way to see additional resource details, which is why I think it makes sense to add it with the appropriate explanation just after we talk about the Resource option in OSIO. The UG is not in any narrative form, so I'm not sure it make sense to move this stuff outside the "chapter" it's currently in, which talks about all the options available in the Profile and Settings options. Thoughts? We can discuss it in a meeting if that makes it easier.

Reply:
Sounds good, we could meet tomorrow in lieu of yesterday's cancelled meeting. @rkratky would be nice to get your opinion on this as well as #388 , can we have a team meeting tomorrow in lieu of the cancelled one yesterday? We could discuss the prod stuff too.

Reply:
@Preeticp, tomorrow, as in Wednesday 06/20?

Reply:
Yes @rkratky , is 06/20 difficult for you?

Reply:
@Preeticp, no. Scheduling for usual time.